test-util/autorestored-sandbox.js

```diff
@@ -0,0 +1,7 @@
+function autorestoredSandbox() {
```

Comment:
This is no longer needed to have separately, as it's a part of @springworks/test-harness 1.3.0